KizGetdown.html

```diff
@@ -66,16 +72,82 @@
 
       <!-- Header -->
 
-      <header class="header-kizgetdown">
-        <div class="overlay-kizgetdown">
+      <header class="header header-kizgetdown">
+        <div class="overlay overlay-kizgetdown">
           <div class="container">
             <h1 class="hero-title kizgetdown-title">FRIDAY KIZOMBA PARTY</h1>
           </div>
         </div>
       </header>
 
+
+
+<!-- About KizGetdown Section-->
+<section>
+  <div class="kizgetdown-sec bg-1">
+      <div class='container'>
+        <div class="text text-center">
+        <h1>Where to Dance? </h1>
+              <p>The Kiz Getdown! It is thanks to this Kiz community that we’ve been able to level up our musical skills, grow as dancers and as individuals. Well now it is finally time for us to give back to you! 🔥🔥🔥 🚒 🔥 🌮🔥</p>
+              <p>We’re happy and proud to announce our bi-weekly (twice a month) Kiz Social, THE KIZ GETDOWN! 🔥🔥 So come on down, and get down, like a kiz clown…we ain’t messing around. Get down to all kinds of Kiz. DJ Cognition and DJ Art will be dropping the filth (and swag)</p>
+              
+            </div>
+      </div>
+    </div>
+  </section>
+
+<!-- Location Section-->
+</section>
+<div class= "location bg-2">
+<div class = "container">
+  <div class ="text text-left">
```

Comment:
Has the VSCode linter been working? The white spaces should not exist. The attribute and value should always be together. e.g. `class="text"`

And take special attention to the code indentation. It's easier to read/understand and catch mistakes.

Reply:
I meant to do two separate classes over there, "text" and "text-left' , can I not do it that way?

Reply:
the problem here is the extra space between class and =.
```
// bad
<div class = "button"

// correct
<div class="button"
```